src/desktop/apps/article/components/__tests__/InfiniteScrollNewsArticle.test.js

```diff
@@ -67,10 +65,13 @@ describe("<InfiniteScrollNewsArticle />", () => {
       marginTop: "50px",
       isMobile: false,
     }
+
+    clock = sinon.useFakeTimers()
   })
 
   afterEach(() => {
     window.history.replaceState.reset()
+    clock.restore()
```

Comment:
ooh, was this the missing piece?

Reply:
Yup!! `sinon.useFakeTimers()` overrides the global `setTimeout` function.